src/docfx/lib/HashUtility.cs

```diff
@@ -54,6 +55,16 @@ public static string GetFileSha1Hash(string fileName)
             }
         }
 
+        public static string GetSortedMonikerListHash(List<string> monikers)
```

Comment:
naming: `public static string GetMd5HashShort(List<string> items)`. Feels it more suitable in `HashUtility`